experimental/traffic-portal/src/app/core/topologies/topology-details/topology-details.component.html

```diff
@@ -0,0 +1,45 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card appearance="outlined" class="page-content single-column">
+	<tp-loading *ngIf="!topology"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="topology">
```

Comment:
This `ngIf` directive will never evaluate to `false`

Reply:
Removed the `ngIf`, but again, a lot of code got merged that does this:

```
$ git grep '<form ngNativeValidate.*ngIf' src/app/core
src/app/core/cache-groups/asns/detail/asn-detail.component.html:        <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="asn">
src/app/core/cache-groups/cache-group-details/cache-group-details.component.html:       <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="cacheGroup">
src/app/core/cache-groups/coordinates/detail/coordinate-detail.component.html:  <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="coordinate">
src/app/core/cache-groups/divisions/detail/division-detail.component.html:      <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="division">
src/app/core/cache-groups/regions/detail/region-detail.component.html:  <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="region">
src/app/core/cdns/cdn-detail/cdn-detail.component.html: <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="cdn">
src/app/core/parameters/detail/parameter-detail.component.html: <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="parameter">
src/app/core/profiles/profile-detail/profile-detail.component.html:     <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="profile">
src/app/core/servers/capabilities/capability-details/capability-details.component.html: <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="capability">
src/app/core/servers/phys-loc/detail/phys-loc-detail.component.html:    <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="physLocation">
src/app/core/types/detail/type-detail.component.html:   <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="type">
src/app/core/users/roles/detail/role-detail.component.html:     <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="role">
src/app/core/users/tenants/tenant-details/tenant-details.component.html:        <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="tenant">
src/app/core/users/user-details/user-details.component.html:    <form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="user">
```